src/govuk/components/exit-this-page/exit-this-page.mjs

```diff
@@ -0,0 +1,404 @@
+/* eslint-disable es-x/no-function-prototype-bind -- Polyfill imported */
+
+import { normaliseDataset } from '../../common/normalise-dataset.mjs'
+import { nodeListForEach, mergeConfigs, extractConfigByNamespace } from '../../common.mjs'
+import { I18n } from '../../i18n.mjs'
+import '../../vendor/polyfills/Element/prototype/classList.mjs'
```

Comment:
Can you just double check if we need this?

Other components that use `addEventListener()` import it

If a service team JUST imports `exit-this-page.mjs`, do they have everything they need polyfilled?

```suggestion
import '../../vendor/polyfills/Element/prototype/classList.mjs'
import '../../vendor/polyfills/Event.mjs' // addEventListener, event.target normalization and DOMContentLoaded
```

Reply:
Let's import it just in case. Great spot.